src/components/date_picker/date_picker.tsx

```diff
@@ -39,6 +39,19 @@ export const euiDatePickerDefaultTimeFormat = 'hh:mm A';
 
 const DatePicker = _ReactDatePicker as typeof ReactDatePicker;
 
+export type EuiDatePickerPopperPlacement =
```

Comment:
Can you auto-pull these from the lib instead of duplicating? Will something like this work:
```suggestion
export type EuiDatePickerPopperPlacement = ReactDatePickerProps['popperPlacement'];
```

Reply:
When I import from the lib the props tab don't expose the right props. It gets `any`.

<img width="940" alt="Screenshot 2020-04-21 at 12 34 12" src="https://user-images.githubusercontent.com/2750668/79861710-b4e9a700-83cc-11ea-8c4d-e15d43152e4a.png">

When I do this way the documentation exposes the right props:

``` js
export type EuiDatePickerPopoverPlacement =
  | 'bottom'
  | 'bottom-end'
  | 'bottom-start'
  | 'left'
  | 'left-end'
  | 'right'
  | 'right-end'
  | 'right-start'
  | 'top'
  | 'top-end'
  | 'top-start';
```

<img width="951" alt="Screenshot 2020-04-21 at 12 32 29" src="https://user-images.githubusercontent.com/2750668/79861869-06923180-83cd-11ea-800e-46c76a952bd4.png">

Reply:
Yeah we've got some issues when pulling in enum props from other sources in our props table. One of the GSOC projects will be geared towards fixing this. I think, at the moment, it is more important to ensure that it stays aligned with the library and reduce duplicate code. The way we've gone around it is to add some hints in the props comment. You could do something like:

`Position of the popover, ex: 'left', 'right-end', 'top-start'`